[DOC-1426] Docs: Release notes for 2026.1.1.0-b90 (#32857)
yugabyte/yugabyte-db: docs/content/stable/reference/configuration/yb-tserver.md

Enable or disable the query planner's use of batched nested loop join.

> ##### yb_enable_advanced_index_cond_fold

Author: cc @jasonyb

{{% tags/wrap %}}

Default: `true`
{{% /tags/wrap %}}

> Enables advanced folding of multiple conditions on the same index column when binding an index scan to DocDB. When enabled, YugabyteDB can tighten inequality bounds across scan keys, intersect IN arrays, and detect contradictions at bind time. That reduces rows fetched from storage and PostgreSQL-side index rechecks.

Reviewer: ```suggestion
Enables advanced folding of multiple conditions on the same index column when binding index scan conditions to DocDB. When enabled, YugabyteDB can fold more conditions on the same column at bind time. That reduces rows fetched from storage and PostgreSQL-side index rechecks.
```

Don't want to point out specific examples when there are more. Also, tighten inequality bounds was already true when this is off. Therefore, switched to generic "more".

> For example, when an IN/ANY array and an inequality both apply to the same column, the inequality can cull the array before the scan instead of discarding rows after DocDB returns them.

Reviewer: ```suggestion
For example, when an IN array and an inequality both apply to the same column, the inequality can cull the array before the scan instead of fetching the full array and discarding rows after DocDB returns them.
```

This is an example, so just saying IN is fine. ANY is inaccurate because only ANY= is supported, not ANY> and such.
